src/backend/marsha/core/api/video.py

```diff
@@ -686,3 +686,50 @@ def end_sharing(
         serializer = self.get_serializer(video)
         channel_layers_utils.dispatch_video_to_groups(video)
         return Response(serializer.data)
+
```

Comment:
minor commit message:
When participants have joined a live, their list is lost if browser is reloaded.
When participants asks for joining, their list is lost too.
We need a way to persist those lists.
A video API endpoint has been added to store them, and it has been added to
video serializer.
=>
When participants have joined a live, their list is lost if **the** browser is reloaded.
When participants **ask** for joining, their list is lost too.
We need a way to persist those lists.
A video API endpoint has been added to store them, and it has been added to
video serializer.